include/otf2xx/definition/pre_fwd.hpp

```diff
@@ -131,6 +145,7 @@ namespace definition
     using location_group_property = property<location_group>;
     using system_tree_node_property = property<system_tree_node>;
     using calling_context_property = property<calling_context>;
+    using io_file_property = property<io_file>;
```

Comment:
This looks suspicious, as io_file is polymorphic in OTF2. A property on a polymorphic definition is a new thing, we may end up for this PR to add a KNOWN_ISSUE.